plugins/api-docs/src/index.ts

```diff
@@ -33,3 +33,4 @@ export {
   EntityProvidedApisCard,
   EntityProvidingComponentsCard,
 } from './plugin';
+export type { DefaultApiExplorerPageProps } from './components/ApiExplorerPage';
```

Comment:
The `*` export above should handle this already, right?

Reply:
hm `'/components/index.ts` doesn't export from `/ApiExplorerPage`: https://github.com/backstage/backstage/blob/2997bbe3c4a859447933790c7419010189ac8632/plugins/api-docs/src/components/index.ts#L17-L22

Should I change that file to include the `/ApiExplorerPage` dir instead ?

Reply:
Ah, yep! Thinking of the traceable exports outlined in [this ADR](https://backstage.io/docs/architecture-decisions/adrs-adr004), essentially only `*` exports above the leaf level.